<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1516</width>
    <height>800</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>CANExplorer</string>
  </property>
  <property name="locale">
   <locale language="English" country="UnitedStates"/>
  </property>
  <property name="documentMode">
   <bool>false</bool>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QPushButton" name="connect_button">
      <property name="text">
       <string>Connect</string>
      </property>
     </widget>
    </item>
    <item row="0" column="2">
     <spacer name="horizontalSpacer">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>40</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="1" column="0" colspan="3">
     <widget class="QTabWidget" name="tab_widget">
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="tab">
       <attribute name="title">
        <string>Tab 1</string>
       </attribute>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QTableWidget" name="tableWidget">
          <column>
           <property name="text">
            <string>Time</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Tx/Rx</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Msg Type</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Arbitration ID</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Message Type</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>DLC</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Data Bytes</string>
           </property>
          </column>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QToolButton" name="toolButton">
      <property name="text">
       <string>Record</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1516</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionNew_Connection"/>
    <addaction name="actionRecent_Connections"/>
    <addaction name="actionClose"/>
    <addaction name="separator"/>
    <addaction name="actionSettings"/>
    <addaction name="separator"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionHelp"/>
    <addaction name="separator"/>
    <addaction name="actionWhat_s_new"/>
    <addaction name="actionGetting_Started"/>
    <addaction name="separator"/>
    <addaction name="actionContact_Support"/>
    <addaction name="actionSubmit_Bug_Report"/>
    <addaction name="actionSubmit_Feedback"/>
    <addaction name="separator"/>
    <addaction name="actionShow_logs_in_Files"/>
    <addaction name="actionShow_Current_Logs"/>
    <addaction name="separator"/>
    <addaction name="actionCheck_For_Updates"/>
    <addaction name="actionAbout"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionHelp">
   <property name="text">
    <string>Help</string>
   </property>
  </action>
  <action name="actionCheck_For_Updates">
   <property name="text">
    <string>Check For Updates</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string>About</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
  <action name="actionWhat_s_new">
   <property name="text">
    <string>What's new in CANExplorer</string>
   </property>
  </action>
  <action name="actionGetting_Started">
   <property name="text">
    <string>Getting Started</string>
   </property>
  </action>
  <action name="actionContact_Support">
   <property name="text">
    <string>Contact Support ...</string>
   </property>
  </action>
  <action name="actionSubmit_Bug_Report">
   <property name="text">
    <string>Submit Bug Report ...</string>
   </property>
  </action>
  <action name="actionSubmit_Feedback">
   <property name="text">
    <string>Submit Feedback ...</string>
   </property>
  </action>
  <action name="actionShow_logs_in_Files">
   <property name="text">
    <string>Show logs in Files</string>
   </property>
  </action>
  <action name="actionShow_Current_Logs">
   <property name="text">
    <string>Show Current Logs</string>
   </property>
  </action>
  <action name="actionNew_Connection">
   <property name="text">
    <string>New Connection</string>
   </property>
  </action>
  <action name="actionRecent_Connections">
   <property name="text">
    <string>Recent Connections</string>
   </property>
  </action>
  <action name="actionClose">
   <property name="text">
    <string>Close Connection</string>
   </property>
  </action>
  <action name="actionSettings">
   <property name="text">
    <string>Settings</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>actionExit</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>487</x>
     <y>399</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
